packages/i18next/src/lib/types.ts

```diff
@@ -1,8 +1,35 @@
 import type { Awaitable, NonNullObject } from '@sapphire/utilities';
+import type { WatchOptions } from 'chokidar';
 import type { Guild, Message, MessageOptions, StageChannel, StoreChannel, User, VoiceChannel } from 'discord.js';
 import type { InitOptions, StringMap, TFunctionKeys, TOptions } from 'i18next';
 import type { i18nextFsBackend } from 'i18next-fs-backend';
 
+/**
+ * This option is used to set the HMR options.
+ * @since 2.1.6
+ */
+export interface HMROptions {
+	/**
+	 * HMR status for the i18next plugin.
```

Comment:
```suggestion
	 * HMR status for the i18next plugin.
	 * @default false
```

Technically undefined but then treated as a falsey value so it's better to just specify it as false.